packages/location_permissions/lib/src/permission_enums.dart

```diff
@@ -9,11 +9,13 @@ enum PermissionStatus {
   denied,
   granted,
   restricted,
+  whenInUse,
 }
 
 enum ServiceStatus {
   unknown,
   disabled,
   enabled,
   notApplicable,
-}
+  whenInUse,
```

Comment:
The `whenInUse` state is not applicable for the `ServiceStatus` enum. The `ServiceStatus` enum indicates if the location services are available on the specified platform.